packages/contracts-bedrock/scripts/checks/check-frozen-files.sh

```diff
@@ -1,135 +0,0 @@
-#!/usr/bin/env bash
-set -euo pipefail
-
-# Grab the directory of the contracts-bedrock package.
-SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
Yay!  There are just a few other updates I think we need to fully remove this:
- There are a few references to this in various markdown files
- The bedrocks `Justfile` has a task that references this script
- The circleci config references the bedrock justfile task